gadget/gadget.go

```diff
@@ -218,38 +218,38 @@ type VolumeUpdate struct {
 type DiskVolumeDeviceTraits struct {
 	// each member here is presented in descending order of certainty about the
 	// likelihood of being compatible if a candidate physical device matches the
-	// member. I.e. CachedDevicePath is more trusted than CachedKernelPath is
+	// member. I.e. OriginalDevicePath is more trusted than OriginalKernelPath is
 	// more trusted than DiskID is more trusted than using the MappedStructures
 
-	// CachedDevicePath is the device path in sysfs and in /dev/disk/by-path the
-	// volume was measured and observed at during UC20+ install mode.
-	CachedDevicePath string `json:"cached-device-path"`
+	// OriginalDevicePath is the device path in sysfs and in /dev/disk/by-path
+	// the volume was measured and observed at during UC20+ install mode.
+	OriginalDevicePath string `json:"device-path"`
 
-	// CachedKernelPath is the device path like /dev/vda the volume was
+	// OriginalKernelPath is the device path like /dev/vda the volume was
 	// measured and observed at during UC20+ install mode.
-	CachedKernelPath string `json:"cached-kernel-path"`
+	OriginalKernelPath string `json:"kernel-path"`
 
 	// DiskID is the disk's identifier, it is a UUID for GPT disks or an
 	// unsigned integer for DOS disks encoded as a string in hexadecimal as in
 	// "0x1212e868".
 	DiskID string `json:"disk-id"`
 
 	// Structure contains trait information about each individual structure in
-	// the volume that may be useful in identifying whethere a disk matches a
+	// the volume that may be useful in identifying whether a disk matches a
 	// volume or not.
-	Structure []DiskStructureDeviceTraits `json:"mapped-structures"`
+	Structure []DiskStructureDeviceTraits `json:"structures"`
```

Comment:
s/structures/structure/  that's what I see gadget.Volume doing

Reply:
done